scheduler/roles/slurm/tasks/main.yml

```diff
@@ -8,6 +8,10 @@
   ansible.builtin.include_tasks: validation.yml
   run_once: true
 
+- name: Create DB tasks
```

Comment:
@Cypher-Miller - how are you making sure this runs on the exact db server? Probably you need to use delegate_to: